off_highway_can/package.xml

```diff
@@ -19,10 +19,11 @@
   <test_depend>ament_cmake_copyright</test_depend>
```

Comment:
Good finding, can you additionaly switch from individually listing the linter deps to just `ament_lint_common`? This would align the `can` package to all other packages, e.g. [radar](https://github.com/bosch-engineering/off_highway_sensor_drivers/blob/b017db0cca6c7225327bcb34470696d286969df7/off_highway_radar/package.xml#L25). Somehow it was never updated...
```suggestion
  <test_depend>ament_lint_common</test_depend>
```